src/components/CustomDropdownComp.tsx

```diff
@@ -43,54 +47,82 @@ export default function CustomDropdownComp(props: CustomDropdownComponentProps):
     const [value, setValue] = useState<Option>();
 
     const createOption = (label: string, secondaryLabel: string, id: string, urlstring: string) => ({
-        label: <Label
-            DisplayName={label}
-            UrlString={urlstring}
-            ClassNamePrefix={props.classNamePrefix}
-            EnableAvatar={props.useAvatar}
-            SecondLabel={secondaryLabel}
-        />,
+        label: (
+            <Label
+                DisplayName={label}
+                UrlString={urlstring}
+                ClassNamePrefix={props.classNamePrefix}
+                EnableAvatar={props.useAvatar}
+                SecondLabel={secondaryLabel}
+            />
+        ),
         value: label,
+        // eslint-disable-next-line object-shorthand
         id: id,
         secondLabel: secondaryLabel,
         url: urlstring
     });
 
     useEffect(() => {
-        if (props.defaultValue.status === 'available') {
+        if (props.defaultValue.status === "available") {
             const defaultValue = props.defaultValue.items.map(obj => {
-                let { firstLabel, secondLabel, id, imgUrl }: { firstLabel: string; secondLabel: string; id: string; imgUrl: string; } = getLabelValues(obj);
-                return createOption(firstLabel, secondLabel, id, imgUrl);
-            })
+                const {
+                    firstLabelDefaultValue,
+                    secondLabelDefaultValue,
+                    objIdDefaultValue,
+                    imgUrlDefaultValue
+                }: {
+                    firstLabelDefaultValue: string;
+                    secondLabelDefaultValue: string;
+                    objIdDefaultValue: string;
+                    imgUrlDefaultValue: string;
+                } = getLabelValuesDefault(obj);
```

Comment:
make non-exported (private) interface above prop interface. Now it's barely readable.